expo/src/customize/customize.css

```diff
@@ -0,0 +1,64 @@
+/*
+    variable naming scheme :
+    --filename_element_attribute
+*/
+
+:root {
+    
+    /* primary color scheme */
+    --primary_blue: #00ffff;

```

Comment:
Instead of specifying "blue", "pink", and "red", let's do `--primary`, `--secondary`, and `--tertiary`? And can you try using the Bitcamp colors and see how things look?
![image](https://user-images.githubusercontent.com/14797288/54065277-8cb0e000-41ec-11e9-8f3d-532dfee0d137.png)